src/ngraph/op/add.cpp

```diff
@@ -59,6 +59,20 @@ void op::v0::Add::generate_adjoints(autodiff::Adjoints& adjoints, const OutputVe
     adjoints.add_delta(y, delta);
 }
 
+namespace
+{
+    bool constant_fold_add(Node* add, const Output<Node> in0, const Output<Node> in1)
+    {
+        cerr << "Calling constand_fold_add" << endl;
```

Comment:
Please provide a real example of evaluation for a node with multiple _outputs_, for example opset1::Split.